lib/DateTime.pm

```diff
@@ -1531,6 +1536,7 @@ sub subtract_datetime {
     );
 }
 
+## no critic (Subroutines::ProhibitManyArgs)
```

Comment:
If I change line 1540 to `sub _adjust_for_positive_difference { ## no critic (Subroutines::ProhibitManyArgs)` and then remove line 1539 and run `tidyall lib/DateTime.pm`, it doesn't report any perlcritic problems (but the comment does get tidied on to the next line, which then also does not report any perlcritic problems). The perlcritic docs also say that `## no critic (...)` can come on the same line as is reported in the complaint.